tools/cli/go-whisk/whisk/trigger.go

```diff
@@ -46,6 +46,20 @@ type TriggerListOptions struct {
     Docs            bool            `url:"docs,omitempty"`
 }
 
+type TriggerArray []Trigger
```

Comment:
Ok, I'll try this approach, my initial though was to let entities like action, trigger implements some interface which is convenient for container manipulates etc, but anyway, if you think it's not necessary, let's change it.